README.md

```diff
@@ -134,13 +134,13 @@ bin/natalie test/runner.rb test/natalie/if_test.rb test/natalie/loop_test.rb
 
 The files in `spec/` come from the excellent [ruby/spec](https://github.com/ruby/spec)
 project, which is a community-curated repo of test files that any Ruby
-implementation can use to compare its conformance to what MRI (Matz's Ruby
+implementation can be used to compare its conformance to what MRI (Matz's Ruby
```

Comment:
This change is incorrect. It would have been right if the line started with "implementation", but there is some text before it.